ui/imports/shared/views/chat/ChatContextMenuView.qml

```diff
@@ -202,6 +202,7 @@ StatusMenu {
 
     StatusAction {
         objectName: "clearHistoryMenuItem"
+        id: clearHistoryMenuItem
```

Comment:
I think, as per QML conventions, `id` should always be the first element, even prior the `objectName`

Reply:
Should I shuffle the two lines for all objects in the file or just my addition?

Reply:
You don't have to, but we usually live by the scout principle of "leaving the campground cleaner than we found it", so if you find some things that you can clean up when doing a fix, it's always appreciated when you fix those at the same time.

Reply:
I'll do that here for **master** target, but leave #12026 with just the one line relevant fix.

This way we keep the patch on point for the release branch and minimize risk in general (even if non-existent in this particular case)?

Reply:
Sounds good